packages/common/src/groups/HubGroups.ts

```diff
@@ -160,17 +168,31 @@ export async function createHubGroup(
  */
 export async function fetchHubGroup(
   identifier: string,
-  requestOptions: IHubRequestOptions
+  context: IArcGISContext
```

Comment:
This is also a breaking change that needs accounted for on the OD-UI PR (and major version bump when this merges)